frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText.tsx

```diff
@@ -1,25 +1,32 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { t } from "ttag";
-
-import { color } from "metabase/lib/colors";
-import MetabaseSettings from "metabase/lib/settings";
-import ExternalLink from "metabase/core/components/ExternalLink";
-import Icon from "metabase/components/Icon";
 import TippyPopover from "metabase/components/Popover/TippyPopover";
 import { HelpText } from "metabase-lib/expressions/types";
-import { getHelpDocsUrl } from "./ExpressionEditorTextfield/helper-text-strings";
+import {
+  Container,
+  ExampleBlock,
+  ExampleCode,
+  ExampleTitleText,
+  FunctionHelpCode,
+} from "./ExpressionEditorHelpText.styled";
 
-interface ExpressionEditorHelpTextProps {
-  helpText: HelpText;
+export interface ExpressionEditorHelpTextProps {
+  helpText: HelpText | null;
```

Comment:
We usually default to `undefined` rather than `null`. I think we almost never use `null` for props in such cases